fee_allocator/accounting/distribution.py

```diff
@@ -112,66 +101,36 @@ def re_distribute_incentives(
             # Calculate pool weight:
             pool_weight = _pool_weights[pool_id_to_receive]
             # Calculate incentives to receive
-            to_receive = round(incentives_to_redistribute * pool_weight, 2)
-            to_receive_aura = round(incentives_to_redistribute_aura * pool_weight, 2)
-            to_receive_bal = round(incentives_to_redistribute_bal * pool_weight, 2)
-            # Need to check if aura or bal incentives
```

Comment:
We cut out any concern about aura min here and just distribute according to plan.